std_msgs/Header header

float64 gps_time_week
float64[3] acceleration
float64[3] rotation_rate
uint16 gps_week_number
